pic-sure-auth-ui/src/main/webapp/admin/header/header.hbs

```diff
@@ -1,11 +1,22 @@
 <div class="row header-row">
-	<div class="logo-img col-md-4" style="font-size: x-large;
+	<div class="logo-img col-md-3" style="font-size: x-large;
                                               color: white;
                                               top: 20px;">Management console</div>
     <span class="col-md-2"></span>
-    <a class="col-md-2 btn btn-default header-btn" id="userMgmt-header" href="/userManagement">Users</a>
-    <a class="col-md-2 btn btn-default header-btn" id="cnxn-header" href="/connectionManagement">Connections</a>
-	<span class="col-md-1 header-btn">
+    <a class="col-md-2 btn btn-default header-btn" id="userMgmt-header" href="/roleManagement"
+       {{#not_contains privileges 'ROLE_SUPER_ADMIN'}}style="visibility: hidden"{{/not_contains}}>
+        Super Admin Console
+    </a>
+    <a class="col-md-2 btn btn-default header-btn" id="userMgmt-header" href="/userManagement"
+       {{#not_contains privileges 'ROLE_SYSTEM'}}style="visibility: hidden"{{/not_contains}}>Users
+    </a>
+    <!--
+    <a class="col-md-2 btn btn-default header-btn" id="cnxn-header" href="/connectionManagement"
+       {{#not_contains privileges 'ROLE_SYSTEM'}}style="visibility: hidden"{{/not_contains}}>Connections
+    </a>
+    -->
+    <span class="col-md-1"></span>
+    <span class="col-md-1 header-btn">
 		<a class="btn btn-default help-btn">Help</a>
```

Comment:
Should applicationManagement be in this header, or is it reached a different way?

Reply:
Yes. It is reached in a different way for now